esphome/components/safe_mode/button/safe_mode_button.cpp

```diff
@@ -0,0 +1,25 @@
+#include "safe_mode_button.h"
+#include "esphome/core/hal.h"
+#include "esphome/core/log.h"
+#include "esphome/core/application.h"
+
+namespace esphome {
+namespace safe_mode {
+
+static const char *const TAG = "safe_mode_button";
```

Comment:
Tags should always be separated with a `.`

```suggestion
static const char *const TAG = "safe_mode.button";
```